sdcm/nemesis.py

```diff
@@ -1817,6 +1822,17 @@ def disrupt_repair_streaming_err(self):
         self._set_current_disruption('RepairStreamingErr')
         self.break_streaming_task_and_rebuild(task='repair')
 
+    def disrupt_scrub(self):
+        self.log.debug("Corrupt the sstable file")
+        self._destroy_data_and_restart_scylla(corrupt_by_dd=True)
+
+        self.log.debug("Rebuild sstables by sstablescrub")
+        for ks_cf in get_non_system_ks_cf_list(self.loaders.nodes[0], self.cluster.nodes[0]):
+            scrub_cmd = 'curl -s -X GET --header "Content-Type: application/json" --header ' \
+                        '"Accept: application/json" "http://127.0.0.1:10000/storage_service/keyspace_scrub/{}?skip_corrupted=true"'.format(
```

Comment:
Open yourself a task to go back here and change it to nodetool scrub when the issue for nodetool is fixed.

Reply:
Also for the dtest.

Reply:
Opened a ticket: https://trello.com/c/jBSfwaMZ/1817-switch-to-use-nodetool-scrub-from-api-sct-dtest